references/detection/presets.py

```diff
@@ -6,21 +6,24 @@ def __init__(self, data_augmentation, hflip_prob=0.5, mean=(123., 117., 104.)):
         if data_augmentation == 'hflip':
             self.transforms = T.Compose([
                 T.RandomHorizontalFlip(p=hflip_prob),
-                T.ToTensor(),
+                T.PILToTensor(),
+                T.ConvertImageDtype(),
```

Comment:
What are your thoughts on provideing the same API as the equivalent class on transforms?
```suggestion
                T.ConvertImageDtype(dtype=torch.float),
```